shared/utils/charts.ts

```
@@ -426,3 +426,189 @@ export function getClosedPrDelta(
+  labelHeight?: number
+}) {
+  const drawingAreaTop = Number(drawingArea.top ?? 0)
+  const drawingAreaHeight = Number(
+    drawingArea.height ?? Number(drawingArea.bottom ?? 0) - drawingAreaTop,
+  )
+  const drawingAreaBottom = drawingAreaTop + drawingAreaHeight
+
+  const labels = series
+    .map((serie) => {
+      const lastPlot = Array.isArray(serie.plots) ? serie.plots.at(-1) : null
+      if (!lastPlot) return null
```

ESLint

[error] Expected { after 'if' condition.

(curly)